doc/apiv3-documentation.apib

```diff
@@ -3507,6 +3818,8 @@ Gets a list of users that can watch the work package.
 
     **Required permission:** add work package watchers
 
+    *Note that you will only receive this error, if you are at least allowed to see the corresponding project.*
```

Comment:
No copy pasta, but change pasta.
This endpoint was moved from project to work package and the message was not updated.